notebooks/falcon-inference.ipynb

```diff
@@ -48,7 +48,8 @@
    "outputs": [],
    "source": [
     "# install the dependencies\n",
-    "!pip install huggingface_hub tokenizers sentencepiece -r requirements.txt -q"
+    "!pip install bitsandbytes==0.41.0 huggingface_hub tokenizers sentencepiece -r requirements.txt -q"
+""
```

Comment:
Let's install everything so that users can play with other models in this notebook without worrying about deps

```suggestion
    "!pip install -r requirements-all.txt -q"
```